__tests__/fixtures.js

```diff
@@ -23,11 +23,19 @@ const PAGES = {
   NO_ACCOUNTS: `${FIXTURES_CONST.BASE_URL}/no-accounts`,
   SETTINGS: `${FIXTURES_CONST.BASE_URL}/settings`,
   ADD_ACCOUNT_MNEMONIC: `${FIXTURES_CONST.BASE_URL}/add-account/mnemonic_phrase`,
-  SEND: `${FIXTURES_CONST.BASE_URL}/send`
+  ADD_ACCOUNT_VIEWONLY: `${FIXTURES_CONST.BASE_URL}/add-account/view_only`,
+  SEND: `${FIXTURES_CONST.BASE_URL}/send`,
+  ADD_ACCOUNT: `${FIXTURES_CONST.BASE_URL}/add-account`
 };
 
 const NETWORK_NAME_FIXTURE = 'Ropsten';
 
+const FIXTURE_ETHEREUM = 'Ethereum';
+
+const FIXTURE_VIEW_ONLY_ADDRESS = '0x82D69476357A03415E92B5780C89e5E9e972Ce75';
```

Comment:
Comment explaining why this address has tokens.
We most probably want an account dedicated to e2e to avoid our tests breaking through UAT transfers.